stix_shifter/stix_transmission/src/modules/guardium/guardium_ping.py

```diff
@@ -0,0 +1,29 @@
+from ..base.base_ping import BasePing
+import json
+from .....utils.error_response import ErrorResponder
+
+
+class GuardiumPing(BasePing):
+    def __init__(self, api_client):
+        self.api_client = api_client
+
+    def ping(self):
+        # Not implemented for Guardium
+        response = self.api_client.ping_box()
+        response_code = response.code
+
+        response_dict = json.loads(response.read())
+        #print(response)
+
+        return_obj = dict()
+        return_obj['success'] = False
+
+        if len(response_dict) > 0 and response_code == 200:
+            return_obj['success'] = True
```

Comment:
You only need to return `return_obj['success'] = True` if the ping is successful.